### DISCUSSION TITLE
忘记密码，怎么下指令整个移除重新设定？

### DISCUSSION THREAD

**Discussion by ai-gate** [FranzKafkaYu/x-ui#248] [General]
您好，
不知为什么使用 OP5系统无法进入x-uI界面进行安装，所以只好手动安装，后来进去改了密码，但密码没有确认机制，导至打错也不知道无法进入，所以想要重新删除重装，但，密码及记录档都会在，所以无法重装，试了几个linux指令都无法移除，能否帮忙，谢谢

**Comment by FranzKafkaYu** [comment]
x-ui setting -show true 查看面板信息（包含用户名与密码）

**Comment by FranzKafkaYu** [comment]
> 您好， 不知为什么使用 OP5系统无法进入x-uI界面进行安装，所以只好手动安装，后来进去改了密码，但密码没有确认机制，导至打错也不知道无法进入，所以想要重新删除重装，但，密码及记录档都会在，所以无法重装，试了几个linux指令都无法移除，能否帮忙，谢谢

使用x-ui setting -reset命令进行重置（x-ui指代binary二进制文件）
